mani_skill/envs/tasks/quadruped/quadruped_reach.py

```diff
@@ -63,6 +63,13 @@ def _default_human_render_camera_configs(self):
                 # mount=self.agent.robot.links[0],
             )
         ]
+    
+    @property
```

Comment:
remove this. Typically voxels aren't used for this task anyway and also we don't know if this setting makes sense